docs/2.8.0/docs/canton/usermanual/kms/external_key_storage.rst

```diff
@@ -0,0 +1,23 @@
+..
+   Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates.
+..
+   Proprietary code. All rights reserved.
+
+.. _external_key_storage:
+
+Configure External Key Storage and Usage with a Key Management Service (KMS)
+============================================================================
+
+.. enterprise-only::
+
+Canton can use a KMS to `generate and store the private keys itself`.
+We directly use a KMS to generate and store Canton's private keys
+and then use its API to securely sign an decrypt messages. A Canton node still stores
+the corresponding public keys in its stores so that it can verify signatures and
+encrypt messages without having to rely on the KMS.
```

Comment:
It's not clear who `itself` refers to here. Is it Canton or the KMS? Maybe a better phrasing would be

```suggestion
Canton can delegate all asymmetric decryption and signing operations to a KMS. In that mode, private keys
are stored on the KMS, not in Canton's own storage nor in memory. Decryption and signing operations each
require an API call to the KMS. A Canton node still stores
the corresponding public keys in its stores so that it can verify signatures and
encrypt messages without having to rely on the KMS.
```